libraries/compute-utils/function-views/src/sensitivity-analysis-view.ts

```diff
@@ -86,12 +86,12 @@ export class SensitivityAnalysisView {
     const analysisInputs = {
       analysisType: {
         input: ui.choiceInput(
-          'Method', ANALYSIS_TYPE.GRID_ANALYSIS, [ANALYSIS_TYPE.GRID_ANALYSIS, ANALYSIS_TYPE.RANDOM_ANALYSIS, ANALYSIS_TYPE.SOBOL_ANALYSIS],
+          'Method', ANALYSIS_TYPE.RANDOM_ANALYSIS, [ANALYSIS_TYPE.GRID_ANALYSIS, ANALYSIS_TYPE.RANDOM_ANALYSIS, ANALYSIS_TYPE.SOBOL_ANALYSIS],
```

Comment:
It is better to remove "GRID_ANALYSIS" value at all from enum, since it is easy to forget to comment out it everywhere. E.g. it is still here in the choiceInput's list.

Commenting out here is too complex, it is much better to remove everything completely. If we will need to restore this type of analysis, we always can revert a commit.

Reply:
GRID_ANALYSIS is not removed. Just input for type of the segment grid (since now normal, log, etc. are not implemented) - it's Andrew's suggestion. 
RANDOM_ANALYSIS is used by default - again, Andrew's suggestion.

Reply:
Got it. My suggestion stays the same - it is better to remove code lines at all instead of commenting them out

Reply:
Ok